src/sqlfluff/dialects/dialect_snowflake.py

```diff
@@ -120,13 +120,6 @@
         name="warehouse_size",
         type="warehouse_size",
     ),
-    DoubleQuotedLiteralSegment=NamedParser(
-        "double_quote",
-        CodeSegment,
-        name="quoted_literal",
-        type="literal",
-        trim_chars=('"',),
-    ),
```

Comment:
This isn't part of Snowflake and isn't used anywhere in the dialect so cleaning up.